sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -67,11 +67,13 @@ static const struct {
     { "iphc", NETOPT_6LO_IPHC },
     { "preload", NETOPT_PRELOADING },
     { "promisc", NETOPT_PROMISCUOUSMODE },
+    { "phy_busy", NETOPT_PHY_BUSY },
     { "raw", NETOPT_RAWMODE },
     { "rtr_adv", NETOPT_IPV6_SND_RTR_ADV },
     { "iq_invert", NETOPT_IQ_INVERT },
     { "rx_single", NETOPT_SINGLE_RECEIVE },
-    { "chan_hop", NETOPT_CHANNEL_HOP }
+    { "chan_hop", NETOPT_CHANNEL_HOP },
+    { "checksum", NETOPT_CHECKSUM }
```

Comment:
does the compilation fail with a trailing comma?

Reply:
Nope, it doesn't. Should I add one?

Reply:
yes please, always having a trailing comma makes rebasing easier and less potential merge conflicts when two people change the same list

Reply:
Done, took the liberty to squash directly.